Makefile

```diff
@@ -61,6 +61,15 @@ install: init
 build: init
 	./scripts/build.sh
 
+.PHONY: build-go
+build-go:
+	go build ./cmd/swapd
+	go build ./cmd/swapcli
+
+.PHONY: build-go-darwin
+build-go-darwin:
+	GOOS=darwin GOARCH=arm64 $(MAKE) build-go
```

Comment:
I'm a little confused.  `build`, `build-all`, `build-go` and `build-go-darwin` are all go specific, but the `build-*` targets depend on `init` which builds the rust code. Which targets are supposed to be setting `cgodleq`?

Reply:
yeah, I agree this is confusing, I updated the tags so `cgodleq` is default again, and `build-go` and `build-go-darwin` use `-tags=fakedleq`

Reply:
I don't see anyone setting `-tags=fakedleq`. Did you push that change? If you want me to set an environment variable here in the Makefile and update the script to use it, Iet me know.

Reply:
it didn't push for some reason, updated